efficientdet/det_model_fn.py

```diff
@@ -582,14 +535,21 @@ def metric_fn(**kwargs):
         tf.tile(tf.expand_dims(box_loss, 0), [
             params['batch_size'],
         ]), [params['batch_size'], 1])
+
+    params['nms_configs']['max_nms_inputs'] = anchors.MAX_DETECTION_POINTS
+    detections_bs = postprocess.generate_detections(params, cls_outputs,
```

Comment:
@fsx950223 FYI, with this utility, eval doesn't depends on pyfunc anymore.

You may leverage this utility for keras train/eval.

Reply:
LGTM